transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollSocketChannelTest.java

```diff
@@ -355,12 +380,12 @@ public void channelReadComplete(ChannelHandlerContext ctx) throws Exception {
 
         void assertSingleRead() throws InterruptedException {
             assertTrue(latch.await(5, TimeUnit.SECONDS));
-            assertEquals(1, count.get());
+            assertTrue("count: " + count.get(), count.get() >= 1);
```

Comment:
the intention is for the string to give you context as to what the value actually was at the time of failure, otherwise this is not recoverable when you encounter a test failure (you will just see `assertion failed`).